.github/workflows/test.yml

```diff
@@ -0,0 +1,26 @@
+name: Run tests
+
+on: [pull_request]
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        julia-version: ['1.6.7', '1.8.2']
```

Comment:
We can probably set the versions to automatically take the latest LTS version and the newest development build. I think it can be`['1.6' , 'nightly']` but please test.
I can see that other Julia packages use these https://github.com/JuliaMath/Interpolations.jl/actions/runs/3206608573/workflow

https://github.com/yeesian/ArchGDAL.jl/actions/runs/3392197412/workflow

Reply:
I agree, I was just unsure how to refer to LTS version, havn't found anyone using 'lts', only specific version numbers like '1.6'. 'nightly' works I actually removed it because I was unsure what it means. Is that just the newest build of Julia?

Reply:
So I can make your proposed change, no problem :) I just wonder if there is a way to avoid having to change '1.6' manually once the LTS version changed at some point.